integration/test/basic/basic_test.go

```diff
@@ -65,7 +61,7 @@ func TestChartLifecycle(t *testing.T) {
 
 	// Test Update
 	l.Log("level", "debug", "message", fmt.Sprintf("updating %s", cr))
-	err = updateChartOperatorResource(helmClient, cr)
+	err = r.UpdateResource(cr, templates.UpdatedChartOperatorResourceValues, "stable")
```

Comment:
Well, isn't it that `templates.UpdatedChartOperatorResourceValues` is only the template with some weird env var placeholders? So we have to do the env magic here at least and provide the rendered template to the update method.

Reply:
No, in this project it is actually a `values` const - see here: https://github.com/giantswarm/chart-operator/blob/6b47a1b1081a9e73f139b655ab1a12a2ed526cb1/integration/templates/chart_operator_resource_values.go#L15

I started #95 with the intention to use a proper template from `apiextensions` but will drop it as it has gotten a bit stale. The change is still coming though.

Reply:
Ok cool. We are going to make this dynamic later on. I see.